src/components/form/businessCreateProjectForm.tsx

```diff
@@ -52,25 +52,23 @@ const profileFormSchema = z.object({
     )
     .optional(),
 
-  description: z.string().max(160).min(4).optional(),
+  description: z.string().max(500).min(20).optional(),
   profiles: z
     .array(
       z.object({
         domain: z.string(),
-        freelancersRequired: z // condition for freelancer
-          .string()
-          .refine((val) => parseInt(val, 10) > 0, {
-            message: 'Number of freelancers required should be greater than 0.',
-          }),
+        freelancersRequired: z.string().refine((val) => parseInt(val, 10) > 0, {
+          message: 'Number of freelancers required must be greater than 0.',
+        }),
         skills: z.array(z.string()),
         experience: z.string(),
         minConnect: z.string(),
         rate: z //condition for rate
           .string()
           .refine((val) => parseFloat(val) >= 0, {
-            message: 'Per hour rate should not be less than 0.',
+            message: 'Rate per hour must be non-negative.',
           }),
-        description: z.string().max(160).min(4),
+        description: z.string().max(200).min(20),
```

Comment:
why are there 2 description here?

Reply:
one description is for project and one for project-profile